hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/ViewFileSystem.java

```diff
@@ -1107,6 +1107,7 @@ public boolean hasPathCapability(Path path, String capability)
     final long creationTime; // of the the mount table
```

Comment:
You may want to add the similar changes in ViewFs.java  to cover FileContext based fs impl.

Reply:
Added change in ViewFs.java